compiler/ballerina-lang/src/main/java/org/wso2/ballerinalang/compiler/bir/codegen/JvmCodeGenUtil.java

```diff
@@ -158,6 +160,9 @@ public class JvmCodeGenUtil {
     private static final Pattern JVM_RESERVED_CHAR_SET = Pattern.compile("[.:/<>]");
     public static final String SCOPE_PREFIX = "_SCOPE_";
     public static final NameHashComparator NAME_HASH_COMPARATOR = new NameHashComparator();
+    public static Boolean isOptimizedCodegen = false;
```

Comment:
Let's make this consistent with other places where we have used the same term
```suggestion
    public static Boolean isOptimizedCodeGen = false;
```

Reply:
Fixed in eb164e9b0bfd3dac61e1b00a86d30e96f11b33a2. Will look for other places as well.